packages/playwright-test/src/reporters/base.ts

```diff
@@ -122,7 +122,7 @@ export class BaseReporter implements Reporter {
   }
 
   protected generateStartingMessage() {
-    const jobs = Math.min(this.config.workers, this.config._internal.maxConcurrentTestGroups);
+    const jobs = Math.min(this.config.workers, FullConfigInternal.from(this.config).maxConcurrentTestGroups);
```

Comment:
Let's get rid of this by either updating FullConfig or via options as discussed.